tests/Fixer/PhpUnit/PhpUnitShortWillReturnFixerTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Michał Adamski <michal.adamski@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitShortWillReturnFixer
+ */
+final class PhpUnitShortWillReturnFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @var ConfigurableFixerInterface
+     */
+    protected $fixer;
+
+    /**
+     * @param string        $expected
+     * @param string | null $input
+     * @param array         $config
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null, $config = [])
+    {
+        $this->fixer->configure($config);
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideFixCases()
+    {
+        return [
+            'will return with integer' => [
```

Comment:
we need to add cases with multilines and with comments put in ugly places

Reply:
```php
$someMock
    ->method('someMethod')
        ->willReturn(10)
        ;
```

Reply:
```php
$someMock
    ->method('someMethod')
        ->will(
            // foo
            $this->returnValue(10)
            // bar
        );
```

```php
$someMock->method('someMethod')->/* a */will/* b */(/* c */ $this->returnValue(10) /* d */);
```

Reply:
done